Marlin/Marlin_main.cpp

```diff
@@ -596,6 +596,43 @@ void setup_powerhold() {
   #endif
 }
 
+/*
+  Enable 24V to fans, E-reg, Cartridge Holder, Cartridges
+  if no short circuit is detected on line
+  ~200us, blocking time
+  Requires serial to be enabled to generate error condition
+*/
+void enable_power_supply() {
+  // Turn on FAN first so that 24V does not have a path to 24V_SW through the Rambo. TODO: Disconnect these two circuits.
+  pinMode(FAN_CHASSIS_TOP_PIN, OUTPUT);
+  digitalWrite(FAN_CHASSIS_TOP_PIN, HIGH);
+  delay(6);  // 40ms delay for line to settle. NOT NEEDED. Fuse set to 65ms startup delay by default.
+  
+  analogRead(PS_MONITOR_PIN);  // Throw out first reading
+  int V_Monitor_Result = analogRead(PS_MONITOR_PIN);
+  if (V_Monitor_Result > PS_ENABLE_LOWER_LIMIT)   // Test for 24V short to gnd
+  {
+    if (V_Monitor_Result <= PS_ENABLE_UPPER_LIMIT) // Test for 24V short to 5V
+    {
+      digitalWrite(PS_FORCE_ON_LL,LOW); // Pull Power Supply enable low to force on
+      pinMode(PS_FORCE_ON_LL, OUTPUT);  //
+      delayMicroseconds(100);           // 100us minimum delay to register on
+      pinMode(PS_FORCE_ON_LL, INPUT);   // Release Power Supply enable to allow current limiting
+    }
+    else 
+    {
+      // TODO: Add to error handling protocol
+      // TODO: Figure out if coming up from power-down state or reset condition
+      //SERIAL_ECHOLN("ERROR TURNING ON 24V POWER. Detected 24V/5V short... PLEASE CHECK HARDWARE...");
```

Comment:
Because right now it will issue that error every time we reset the processor. Basically, if we enter that state it means we have >= 5V on the 24V_SW line. This means either the line is shorted to 5V, or (more likely) 24V has already been enabled and we are coming up from a reset, not a power-down state. In either case, the action is to NOT enable power.

I want to see if we can make this distinction, but it might require a change to hardware.